packages/graphback-cli/src/commands/generate.ts

```diff
@@ -0,0 +1,11 @@
+import { generate } from '../utils/generateHelper'
```

Comment:
utils? I would say have separate implementation for each command somewhere but utils are not the best place. GenerateCommand maybe as name?

Reply:
How does helpers sound?